src/components/universalSearch/tabs/tabStores/serviceInsightsTabStateStore.js

```diff
@@ -28,7 +28,7 @@ export class ServiceInsightsTabStateStore {
         this.search = search;
 
         // TODO: reconcile this with hasValidSearchProps() in serviceInsights.jsx
-        this.isAvailable = subsystems && enableServiceInsights && search.serviceName;
+        this.isAvailable = !!(subsystems && enableServiceInsights && search.serviceName);
```

Comment:
`this.isAvailable` should be cast to bool (was being set to serviceName string value in some cases)